src/main/java/com/cricketcraft/ftbisland/IslandCreator.java

```diff
@@ -62,7 +67,25 @@ public static boolean spawnIslandAt(World world, int x, int y, int z, String pla
                 for(int c = 0; c < 5; c++ ) {
                     world.setBlock(x, y + c + 1, z, Blocks.log);
                 }
-            } else {
+            // Creates an island of a single block of grass with a sign -- #GrassBlockChallenge from Darkosto
+            } else if (FTBIslands.islandType.equalsIgnoreCase("grass")) {
+        		world.setBlock(x, y, z, Blocks.grass);
+	 			world.setBlock(x, y + 1, z, Blocks.standing_sign, 6 , 3);
+	 			((TileEntitySign) world.getTileEntity(x, y + 1, z)).signText[0] = "You get it yet?";
+	 		// Creates a Garden of Glass island type
+            } else if (FTBIslands.islandType.equalsIgnoreCase("GoG")) {
+        		if (Loader.isModLoaded("Botania")) {
+        			SkyblockWorldEvents.createSkyblock(world,  x,  y -1,  z);
```

Comment:
This may want to be changed to the full import, vazkii.botania.common.world.SkyblockWorldEvents.createSkyBlock(world, x, y -1, z); just to avoid import issues if the mod is not loaded.